.github/workflows/python-package.yml

```diff
@@ -28,4 +28,9 @@ jobs:
         flake8 . --count --max-complexity=10 --max-line-length=127 --statistics
     - name: Test with pytest
       run: |
-        echo Add your testing here
+        pytest -v
+    - name: Publish the package
+      run: |
+        cp .pypirc $HOME/
+        python3 setup.py sdist 
```

Comment:
Nice!
One could also use [build](https://pypi.org/project/build/) for simpler packaging

Reply:
Oh, it's simpler indeed. Found these commands in several tutorials and a documentation. Thanks!